scripts/buildLogos.js

```diff
@@ -40,6 +46,12 @@ async function buildLogos() {
 
   // eslint-disable-next-line no-restricted-globals,compat/compat,promise/no-native
   await Promise.all(logoPromises);
+
+  if (shouldCommit) {
+    await $`git add cdn`;
+    await $`git commit -m "feat: generate CDN packages"`;
```

Comment:
Let's make this a chore to avoid any future issues with minor version bumps. 
```suggestion
    await $`git commit -m "chore: generate CDN packages"`;
```